portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,8 +39,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         PrintWriter writer = response.getWriter();
 
         response.setContentType("text/html;");
-        
-        writer.println("<h1>Hello world!</h1>");
-        writer.println("<h2>Greeting from " + renderedName + "!</h2>");
+        writer.println("Greeting from " + renderedName + "!");
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes 

use single quotes for ordinary string literals

Reply:
Wow! This double quotes thing that I am using comes from C++!

Reply:
Sorry this was for JS. I got confused. The style guides vary a lot by language